documentation/zones.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Zone support in vSphere Cloud Provider
+---
+
+## Introduction
+
+In the vSphere infrastructure, kubernetes nodes are run as VMs on an ESX host and Volumes are created as VMDK files on a datastore. The Volume is always placed on a datastore that is shared among all the node VMs of the k8s cluster. This is done so that the Volume can be attached to a pod scheduled on any of the node VMs. However, this implies that there should be a shared datastore among the node VMs in vSphere infrastructure to create Volumes. It also implies that the datastore capacity of any other non-shared datastores cannot be utilized for creating Volumes.
+
+With the introduction of zones in kubernetes, the cloud provider can express the topology of the cloud infrastructure as zones. Kubernetes nodes and Volumes have labels denoting the zone information. The k8s pod scheduler is zone aware in placing pods on nodes in the required zone.
+
+vSphere Cloud Provider added Zones support since Kubernetes version 1.14.
+
+## Mark Zones and Regions in vCenter
+
+Zones and Regsions are marked by creating and applying Tags in vCenter. The Tags can be applied to a Datacenter, Cluster or ESX Host. Tags are inherited from the parent and can be overridden on the child in this hierarchy. So a Tag applied to a Datacenter is inherited by all Clusters and Hosts in that Datacenter, and a Tag applied to a Cluster overrides any Tags that it might inherit from its parent Datacenter.
```

Comment:
Link to documentation on how to create tag and category would help user 

https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.vcenterhost.doc/GUID-E8E854DD-AA97-4E0C-8419-CE84F93C4058.html

Reply:
Good idea. Added this one.